cloud/docker/docker.py

```diff
@@ -1347,9 +1402,11 @@ def main():
             memory_limit    = dict(default=0),
             memory_swap     = dict(default=0),
             docker_url      = dict(),
-            docker_tls_cert = dict(),
-            docker_tls_key  = dict(),
-            docker_tls_cacert = dict(),
+            use_tls         = dict(default=None, choices=['no', 'encrypt', 'verify']),
+            tls_client_cert = dict(required=False, default=None, type='str'),
```

Comment:
You are correct about the default values.  I like explicitness as it's self documenting (especially for default.  type and required I am more indifferent to).  I can get rid of required and type if you want.